app/backend/resources/schema.edn

```diff
@@ -1817,7 +1817,86 @@
           {:db/ident :land-owner-opinion/authority-position
            :db/valueType :db.type/string
            :db/cardinality :db.cardinality/one
-           :db/doc "Rich text containing the authoritys position on the opinion"}]]}]
+           :db/doc "Rich text containing the authoritys position on the opinion"}]]}
+
+ {:db/ident :migration/thk-contracts
+  :txes [[{:db/ident :thk.contract/number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Link to contract file in document management system for procurement part"}
+
+          {:db/ident :thk.contract/procurement-number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Procurement no /Hanke nr - (if starts with number then converted into link to State procurement registry as https://riigihanked.riik.ee/rhr-web/#/procurement/<proc_No>), if starts with other symbol then leave it as a number)"}
+
+          ;; Contract statuses
+          {:db/ident :thk.contract.status/contract-signed
+           :enum/attribute :thk.contract/status}
+          {:db/ident :thk.contract.status/in-progress
+           :enum/attribute :thk.contract/status}
+          {:db/ident :thk.contract.status/warranty-period
+           :enum/attribute :thk.contract/status}
+          {:db/ident :thk.contract.status/completed
+           :enum/attribute :thk.contract.response/status}
+
+          {:db/ident :thk.contract/status
+           :db/valueType :db.type/ref
+           :db/cardinality :db.cardinality/one
+           :db/doc "Enum reference to contract status"}
+
+          {:db/ident :thk.contract/region
+           :db/valueType :db.type/string
```

Comment:
Change to reference and add regions as enums

Reply:
Could also add the region for project importing.

Reply:
region enums should be common for sharing between projects and contracts. So it can't be thk.contract/region nor thk.project/region. Maybe estonia/region estonia.region/west